### PR TITLE
New device IMCO POWER

### PR DESCRIPTION
**Pull request by Martin22**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/power-supply-imco-power/6623/2

**Comment by PipoCanaja** [comment]
Hi @Martin22 
Is it mandatory to create 2 different OSes ?

File: includes/definitions/discovery/imcopower-ls110.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+mib: IMCO-LS110-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: sdsTempSensValue
+                    num_oid: '.1.3.6.1.4.1.33283.1.30.5.1.3.{{ $index }}'
+                    index: 0
```
Comment (PipoCanaja) [inline includes/definitions/discovery/imcopower-ls110.yaml]: Hi,
You need to define the index like "index: 'sdsTempSensValue.{{ $index }}'" and not specify a value.

**Comment by PipoCanaja** [comment]
Other question, have you checked if the device is compatible with rfc1628 (standard UPS mib) ? 
If yes, you can just add "rfc1628_compat" in the yaml file of the OS :  
```
os: imcopower-big
text: 'Imco Power'
rfc1628_compat: true
<etc>
```

**Comment by Martin22** [comment]
> Hi @Martin22
> Is it mandatory to create 2 different OSes ?

I can merge, but unnecessarily more OID will be tested. Each device has completely different OIDs.
Do you think I should do one device?

**Comment by Martin22** [comment]
> Other question, have you checked if the device is compatible with rfc1628 (standard UPS mib) ?
> If yes, you can just add "rfc1628_compat" in the yaml file of the OS :
> 
> ```
> os: imcopower-big
> text: 'Imco Power'
> rfc1628_compat: true
> <etc>
> ```

Yes. Is not compatible.

**Comment by PipoCanaja** [comment]
All right. So we can keep this. Please check the Travis CI tests and ensure that they are fixed. Basically, the issue right now is with the SVG file. You can use "https://www.svgminify.com/" to improve the file.

**Comment by PipoCanaja** [comment]
please do `git checkout master bootstrap/cache/.gitignore` as well. The file got changed.

**Comment by Martin22** [comment]
> please do `git checkout master bootstrap/cache/.gitignore` as well.
> 
> `git checkout master logs/.gitignore`
> `git checkout master rrd/.gitignore`
> `git checkout master storage/app/.gitignore`
> `git checkout master storage/app/public/.gitignore`
> `git checkout master storage/debugbar/.gitignore`
> etc etc
> and all the other files that are changed in the list ...

Is it okay now?

**Comment by PipoCanaja** [comment]
One last one : 
`bootstrap/cache/.gitignore`

**Comment by Martin22** [comment]
> LGTM

@PipoCanaja I am sorry. Everything should be all right now.

**Comment by Martin22** [comment]
@PipoCanaja I am sorry. Everything should be all right now.

**Comment by Martin22** [comment]
@PipoCanaja 
Please approve again. Thank You

**Comment by PipoCanaja** [comment]
Thanx @Martin22 !

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-63-release-changelog-april-2020/11828/1

**Pull request merged**